apps/ui/src/components/SpaceDelegates.vue

```diff
@@ -460,7 +460,10 @@ watchEffect(() => setTitle(`Delegates - ${props.space.name}`));
         :open="isUndelegating"
         :chain-id="delegation.chainId"
         :execute="undelegateFn"
-        @confirmed="handleUndelegateConfirmed"
+        @confirmed="
+          handleUndelegateConfirmed();
+          isUndelegating = false;
+        "
```

Comment:
Not important but instead of here, better to move `isUndelegating = false;` inside `handleUndelegateConfirmed`